### DISCUSSION TITLE
audio example

### DISCUSSION THREAD

**Discussion by WojciechGw** [picocomputer/community#71] [Q&A] [answered]
refers to file ezpsg.h

Should the enumeration bb0 be here ?

```
19 enum ezpsg_notes
20 {
21    // TODO add all flat notes
22    a0 = 0,
23    as0 = 1,
24    bb0 = 1,
25    b0 = 2,
```

```
19 enum ezpsg_notes
20 {
21   // TODO add all flat notes
22    a0 = 0,
23    as0 = 1,
24    b0 = 2,
```
It seems to me that if so, the definition of EZPSG_NOTE_FREQS is missing the repetition of the frequency value 29

instead of
`113 #define EZPSG_NOTE_FREQS 28, 29, 31, 33, 35, 37, 39, 41,                 \`

should be
`113 #define EZPSG_NOTE_FREQS 28, 29, 29, 31, 33, 35, 37, 39, 41,                 \`

Am I wrong ?

**Comment by rumbledethumps** [comment, marked as answer]
Both are the same note with the same enum value:
as0 **= 1**, # A sharp 0
bb0 **= 1**, # B flat 0
So the code is correct.

**Reply by WojciechGw** [reply]
thank You
